Explorer/Assets/DCL/Interaction/Systems/WritePointerEventResultsSystem.cs

```diff
@@ -88,7 +88,7 @@ private void WriteResults([Data] Vector3 scenePosition, ref PBPointerEvents pbPo
 
         private void AppendMessage(CRDTEntity sdkEntity, RaycastHit sdkHit, InputAction button, PointerEventType eventType)
         {
-            ecsToCRDTWriter.AppendMessage<PBPointerEventsResult, (RaycastHit sdkHit, InputAction button, PointerEventType eventType, ISceneStateProvider sceneStateProvider)>(
+            ecsToCRDTWriter.AppendMessage<PBPointerEventsResult, (RaycastHit? sdkHit, InputAction button, PointerEventType eventType, ISceneStateProvider sceneStateProvider)>(
```

Comment:
maybe we shouldn't use tuples here?
It's a reference type

Reply:
https://www.tutorialsteacher.com/csharp/valuetuple